src/providers/pdal/qgspdaleptgenerationtask.cpp

```diff
@@ -148,47 +151,8 @@ QString QgsPdalEptGenerationTask::guessUntwineExecutableBinary() const
   return QString( untwineExecutable );
 }
 
-QString QgsPdalEptGenerationTask::outputDir() const
+QString QgsPdalEptGenerationTask::outputFile() const
 {
-  return mOutputDir;
+  return mOutputFile;
 }
 
-bool QgsPdalEptGenerationTask::prepareOutputDir()
-{
-  const QFileInfo fi( mOutputDir + "/ept.json" );
-  if ( fi.exists() )
-  {
-    QgsMessageLog::logMessage( tr( "File %1 is already indexed" ).arg( mFile ), QObject::tr( "Point clouds" ), Qgis::MessageLevel::Info );
-    return true;
-  }
-  else
-  {
-    if ( QDir( mOutputDir ).exists() )
-    {
-      if ( !QDir( mOutputDir ).isEmpty() )
-      {
-        if ( QDir( mOutputDir + "/temp" ).exists() )
-        {
-          QgsMessageLog::logMessage( tr( "Another indexing process is running (or finished with crash) in directory %1" ).arg( mOutputDir ), QObject::tr( "Point clouds" ), Qgis::MessageLevel::Warning );
-          return false;
-        }
-        else
-        {
-          QgsMessageLog::logMessage( tr( "Folder %1 is non-empty, but there isn't ept.json present." ).arg( mOutputDir ), QObject::tr( "Point clouds" ), Qgis::MessageLevel::Critical );
-          return false;
-        }
-      }
-      else
-      {
-        // untwine expects that the output directory does not exist at all
-        if ( !QDir().rmdir( mOutputDir ) )
-        {
-          QgsMessageLog::logMessage( tr( "Failed to remove empty directory %1" ).arg( mOutputDir ), QObject::tr( "Point clouds" ), Qgis::MessageLevel::Critical );
-          return false;
-        }
-      }
-    }
-  }
-
-  return true;
-}
```

Comment:
are you sure we do not need any of this? e.g. how about the check if the file is already indexed? or that there is indexing already running? (maybe in a different qgis instance)